File: #03/Categoria 03/q5 - gangorra/gangorra.hs
gangorra :: (Ord a, Num a, Num p) => a -> a -> a -> a -> p
gangorra p1 c1 p2 c2  = 
    if (p1 * c1) > (p2 * c2)
        then -1
    else if (p1 * c1) < (p2 * c2)
        then 1
    else 0


main :: IO ()
main = do
    a <- readLn :: IO Int
    b <- readLn :: IO Int
    c <- readLn :: IO Int
    d <- readLn :: IO Int
    print $ gangorra a b c d

File: #03/Categoria 03/q5 - gangorra/gangorra.js
gangorra =  (p1,c1,p2,c2) => {
    if ((p1 * c1) > (p2 * c2)) return -1
    else if((p1 * c1) < (p2 * c2)) return 1
    else return 0
}

console.log(gangorra(30 ,100 ,60 ,50));
console.log(gangorra(40, 40, 38, 60,));
console.log(gangorra(35, 80, 35, 75,));